src/markdown_tables.jl

```diff
@@ -40,7 +40,7 @@ Keyword arguments:
 """
 function markdown_table(io :: IO, df :: DataFrame; hl=passfail_highlighter(df), kwargs...)
   header, table = format_table(df, MDformat; kwargs...)
-  pretty_table(io, table, header, tf=markdown, highlighters=hl)
+  pretty_table(io, table, header, tf=tf_markdown, highlighters=hl)
```

Comment:
What is tf_markdown?

Reply:
In the new version of PrettyTables they rename the constant `markdown` as `tf_markdown`